src/settings/SettingsSelfExclusion.js

```diff
@@ -68,13 +68,19 @@ export default class SettingsSelfExclusion extends PureComponent {
 			exclude_until,
 			timeout_until,
 		};
+
 		try {
 			await api.setSelfExclusion(newSelfExclusionSettings);
 			this.setState({
 				success: true,
 				serverError: false,
 			});
-			setTimeout(() => this.setState({ success: false }), 3000);
+			setTimeout(() => {
```

Comment:
@qingweibinary would actually changed it by updating the live-api call as mentioned on the another on trello. I am thinking of making the call returns with api error response from the liveapi instead of throwing from live api. So the message here was just temporary pending the changes from live-api. I would use ErrorCode or something similar.